cli/commands/start.go

```diff
@@ -2,13 +2,28 @@ package commands
 
 import (
 	"fmt"
+	"time"
 
 	"github.com/apex/log"
 	"github.com/spf13/cobra"
 
+	"github.com/tarantool/cartridge-cli/cli/project"
 	"github.com/tarantool/cartridge-cli/cli/running"
 )
 
+const (
+	defaultStartTimeout = 1 * time.Minute
+)
+
+var (
+	daemonize      bool
+	timeout        time.Duration
```

Comment:
`type Duration int64` that means that `timeout` is signed.

Could we pass some negative value?

Reply:
This option accepts value in the `00h4m0.5s` form, the negative value will be rejected by cobra (or, as I understood, by the `time.Duration` constructor).

Reply:
I've added error on negative timeout.